Detectors/TOF/workflow/src/CompressedDecodingTask.cxx

```diff
@@ -162,6 +164,11 @@ void CompressedDecodingTask::endOfStream(EndOfStreamContext& ec)
 void CompressedDecodingTask::headerHandler(const CrateHeader_t* crateHeader, const CrateOrbit_t* crateOrbit)
 {
   if (mConetMode) {
+    if (mNCrateOpenTF == 0) {
```

Comment:
Is it safe to rely on the mNCrateOpenTF, and in general, on the rdhHandler? The delivery of every HBF is not guaranteed, so the 1st HBF RDH bearing the o2::trigger::TF might be missed so the mNCrateOpenTF will not be incremented. 
Also, here: https://github.com/AliceO2Group/AliceO2/blob/5b91ca11bd23384b0c742d6b2f8b3c5292270570/Detectors/TOF/workflow/src/CompressedDecodingTask.cxx#L343-L347
For the `mInitOrbit` you can switch to the `DataHeader::firstTForbit`

Reply:
I kept mNCrateOpenTF counters but I removed the check on that for posting data (but the conet mode).
 mInitOrbit = RDHUtils::getHeartBeatOrbit(rdhr); removed as well